src/ocm/components/clusterConfiguration/HostInventory.tsx

```diff
@@ -27,7 +27,7 @@ import { selectCurrentClusterPermissionsState } from '../../selectors';
 import PlatformIntegration from './platformIntegration/PlatformIntegration';
 
 const schedulableMastersTooltip =
-  'This toggle will be "On" and not editable when less than 5 hosts were discovered';
+  'Workloads must be run on control plane nodes when less than 5 hosts are discovered';
```

Comment:
shouldnt this be 
```suggestion
  'Workloads must run on control plane nodes when less than 5 hosts are discovered';
```

?

Reply:
Both variants would be correct, I think. But this is what [Allison suggested](https://docs.google.com/document/d/1b6DzUHDgEIWthqBetLRdfZ_tg8yK5V-Kry7zPSttauE/edit?disco=AAAAtbFA-TM) so let's leave it at that.